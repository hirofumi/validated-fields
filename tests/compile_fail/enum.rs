use validated_fields::ValidatedFields;

#[derive(ValidatedFields)]
enum Enum {
    X,
    Y,
}
