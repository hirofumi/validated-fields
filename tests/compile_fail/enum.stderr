error: ValidatedFields is not supported for enums
 --> tests/compile_fail/enum.rs:4:1
  |
4 | enum Enum {
  | ^^^^

error[E0601]: `main` function not found in crate `$CRATE`
 --> tests/compile_fail/enum.rs:7:2
  |
7 | }
  |  ^ consider adding a `main` function to `$DIR/tests/compile_fail/enum.rs`
